tests/cli/test_install_tar.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+# Note: execute this file from the project root directory
+
+#####
+#
+# Builds tar image and install it using liveimg kickstart command
+#
+#####
+
+set -e
+
+. /usr/share/beakerlib/beakerlib.sh
+. $(dirname $0)/lib/lib.sh
+
+CLI="${CLI:-./src/bin/composer-cli}"
+QEMU_BIN="/usr/bin/qemu-system-$(uname -m)"
+QEMU="$QEMU_BIN -machine accel=kvm:tcg"
+
+
+rlJournalStart
+    rlPhaseStartSetup
+        if ! rlCheckRpm httpd; then
+            dnf -y install httpd
+        fi
+        systemctl start httpd
+
+        ks_path="/var/www/html/ks-tar.cfg"
+        cat > $ks_path << __EOF__
+lang en_US.UTF-8
+timezone America/New_York
+keyboard us
+rootpw redhat
+bootloader --location=mbr
+zerombr
+clearpart --initlabel --all
+autopart
+
+liveimg --url http://10.0.2.2/root.tar.xz
```

Comment:
Is this IP address a stable interface ? With libvirtd on RHEL the default is 192.168.122.1, never seen this one. Maybe you can also grab the actual address of the system and use it (if it is not overly complicated). Whichever one is good as long as we know it is stable.

Reply:
`file:///` urls work as well.

Reply:
What do you mean by that ? If I understand you correctly the tarball image will need to be made available via partition on the hard-drive before we do the installation. Not really easier than firing up httpd and getting the IP address right.

Reply:
Heh, sorry for the brevity. I mistook the for a kickstart we're running from the host, in which case absolute `file://` urls would have worked. This one is passed to a booted system though, so your approach is totally valid. Thanks!